docs/ref/models/querysets.txt

```diff
@@ -647,11 +645,6 @@ It is an error to pass in ``flat`` when there is more than one field.
 If you don't pass any values to ``values_list()``, it will return all the
 fields in the model, in the order they were declared.
 
-Note that this method returns a ``ValuesListQuerySet``. This class behaves
```

Comment:
should we have a versionchanged note to note the historical existence of `ValuesListQuerySet`?